demos/http/common/src/presigned_urls_gen.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+
+import boto3
+from botocore.client import Config
+import argparse
+
+def get_presigned_urls(bucket_name, key_name):
```

Comment:
Let's increase the quality of our python scripts and add function doc strings. 

```suggestion
def get_presigned_urls(bucket_name, key_name) -> None:
    """
    Prints the presigned GET and PUT URLs assigned to the demo specific C macros,
    for the given object key in the given S3 bucket. The URLs are presigned with AWS's
    Signature Version 4.
    Args:
        bucket_name (str): S3 bucket
        key_name (str):  S3 object key
    """
        """
```

Reply:
Sure, sounds good. Will go through and update the script.